pt/deploy/README.md

```diff
@@ -193,14 +194,12 @@ Assim como fez em seu próprio computador, você pode criar um virtualenv na Pyt
 
     20:20 ~ $ source myvenv/bin/activate
 
-    (mvenv)20:20 ~ $  pip install django whitenoise
+    (mvenv)20:20 ~ $  pip install -r requirements.txt
```

Comment:
I might be missing something. But I'm willing to ask, where is the `requirements.txt` file? I believe that, at this point, there ain't none. It's yet to be created at some further step described below. What am I missing?